### PR TITLE
Remove supervise for debug

### PR DESCRIPTION
**Pull request by kentwills**
We don't need to supervise a process running in the foreground.

### CODE REVIEW

File: tests/spec/debug.py [hunk 1]
```diff
@@ -70,25 +70,3 @@ def it_first_stops_the_background_service_if_running():
...
-    wait_for(lambda: assert_svstat('playground/unreliable', state='up'))
-
-    check_call(('pgctl-2015', 'stop'))
-    stdout, stderr = proc.communicate()
-    stdout, stderr = stdout.decode('UTF-8'), stderr.decode('UTF-8')
-
-    assert '''\
-[pgctl] Stopping: unreliable
-[pgctl] Stopped: unreliable
-pgctl-poll-ready: disabled during debug -- quitting
```
Comment (bukzor) [inline tests/spec/debug.py]: This is still a necessary behavior.

Un-delete.

File: pgctl/poll_ready.py [hunk 2]
```diff
@@ -81,8 +81,6 @@ def main():
         # run the wrapped command in the main process
         from sys import argv
         exec_(argv[1:])  # never returns
-    elif os.environ.get('PGCTL_DEBUG'):
-        print_stderr('pgctl-poll-ready: disabled during debug -- quitting')
```
Comment (bukzor) [inline pgctl/poll_ready.py]: ... This is still a necessary behavior.

Un-delete

**Comment by bukzor** [comment]
You seem to have confused `s6-supervise` with `pgctl-poll-ready`.
Since they have different names you should assume they're different things.

**Comment by kentwills** [comment]
@bukzor :p

**Comment by kentwills** [comment]
@bukzor we are no longer supervising the process and it is a long running exec'd process, how do we check the stdout and stderr without getting stuck during proc.communicate()

**Pull request merged**